docs/content/latest/releases/whats-new/latest-releases.md

```diff
@@ -16,11 +16,110 @@ isTocNested: true
 showAsideToc: true 
 ---
 
-Included here are the release notes for all releases in the v2.5 latest release series.
+{{< note title="Notable features and changes" >}}
 
-## Notable features and changes (cumulative for the v2.5 latest release series)
+Included here are the release notes for all releases in the v2.5 latest release series. Content will be added as new notable features and changes are available in the patch releases of the v2.5 latest release series.
 
-Note: Content will be added as new notable features and changes are available in the patch releases of the v2.5 latest release series.
+{{< /note >}}
+
+## v2.5.3 - Mar 9, 2021
+
+**Build:** `2.5.3.1-bxx`
+
+### Downloads
+
+<a class="download-binary-link" href="https://downloads.yugabyte.com/yugabyte-2.5.3.1-darwin.tar.gz">
+  <button>
+    <i class="fab fa-apple"></i><span class="download-text">macOS</span>
+  </button>
+</a>
+&nbsp; &nbsp; &nbsp;
+<a class="download-binary-link" href="https://downloads.yugabyte.com/yugabyte-2.5.3.1-linux.tar.gz">
+  <button>
+    <i class="fab fa-linux"></i><span class="download-text">Linux</span>
+  </button>
+</a>
+<br />
+
+### Docker
+
+```sh
+docker pull yugabytedb/yugabyte:2.5.3.1-bxx
+```
+
+### New Features
+
+#### Yugabyte Platform
+
+* [[7126](https://github.com/yugabyte/yugabyte-db/issues/7126)] PITR: Introduce snapshot schedule
```

Comment:
this is core db work, it's a building block for PITR (point in time restore)

also, not sure how to group this (new feature vs improvement), technically, PITR will be a net new feature, but right now it's a WIP / covers a small subset of usage